dist/obis.js

```diff
@@ -3998,6 +3998,7 @@ jQuery.extend( obis, {
 
 		return (
 			statement.type.replace( /[^a-zA-Z]/g, '_' ) + '-' +
+			('0' + (statement.date..getMonth()+1)).slice(-2) + '-' +
```

Comment:
Nice contribution, thank you! I guess it's necessary now that it's impossible to download a whole year, as the previous web-site allowed!

But before pulling-in, can you explain the double-dot please? `..` Also, if the month is 10-12, will it be rendered as `010`, `011`, `012`?